### PR TITLE
revamp validator manager docs

### PR DESCRIPTION
**Pull request by cam-schultz**
## Why this should be merged
Revamps the Validator Manager docs to be more intuitive to understand. The top-level README provides a brief overview, but the bulk of it is used to describe how end-users interact with the contracts. Warp message specifications are moved to a separate reference document, as is the state transition diagram.

Documents changes made in https://github.com/ava-labs/teleporter/pull/561 and https://github.com/ava-labs/teleporter/pull/573

## How is this documented

### CODE REVIEW

**Review by iansuvak — COMMENTED** [review]
I love the re-org, extra details and diagrams. Just a few minor questions

File: contracts/validator-manager/StateTransition.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## State Transitions
```
[the remark this answers is not in the document]
Comment (cam-schultz) [reply contracts/validator-manager/StateTransition.md:1]: Added a `PoSValidatorManager` section to `README.md` and linked the diagram there.

File: contracts/validator-manager/README.md [hunk 2]
```diff
@@ -1,179 +1,134 @@
...
 
-### `ValidationUptimeMessage`
+The `RegisterSubnetValidatorMessage` is delivered to the P-Chain as the Warp message payload of a [`RegisterSubnetValidatorTx`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#registersubnetvalidatortx). Please see the transaction [specification](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#step-2-issue-a-registersubnetvalidatortx-on-the-p-chain) for validity requirements. The successful `RegisterSubnetValidatorTx` results in a [`SubnetValidatorRegistrationMessage`](./MessageSpec.md#subnetvalidatorregistrationmessage) Warp message indicating that the specified Validator was successfully registered on the P-Chain.
```
[the remark this answers is not in the document]
Comment (cam-schultz) [reply contracts/validator-manager/README.md]: That's correct. I've reworded this section for clarity.

File: contracts/validator-manager/README.md [hunk 3]
```diff
@@ -1,179 +1,138 @@
...
 
-Disabled Subnet validators can re-activate at any time by increasing their balance with an `IncreaseBalanceTx`. Anyone can call `IncreaseBalanceTx` for any validator on the P-Chain. A disabled validator can only be totally removed from the validator set by a call to `initializeEndValidation`.
+### PoAValidatorManager
 
-## Warp Message Format Reference
+Proof-of-Authority Validator management is provided via `PoAValidatorManager`, which restricts modification of the Validator set to an owner address. After deploying `PoAValidatorManager.sol` and a proxy, the `initialize` function takes the owner address, in addition to standard `ValidatorManagerSettings`.
 
-### `RegisterSubnetValidatorMessage`
+### PoSValidatorManagerr
```
Comment (iansuvak) [inline contracts/validator-manager/README.md:57]:
```suggestion
### PoSValidatorManager
```

**Review by iansuvak — APPROVED** [review]

**Review by michaelkaplan13 — APPROVED** [review]

**Pull request merged**